ios/browser/svg/BUILD.gn

```diff
@@ -19,6 +19,7 @@ source_set("svg") {
   ]
 
   sources += [
+    "//third_party/skia/include/codec/SkPixmapUtils.h",
```

Comment:
@Brandon-T can we add a dep for this instead of adding the file or will that cause problems?

Reply:
We need it like this since we can't import Skia anything as a dep on iOS :(